locale/ar_IQ/locale.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/themes/bootstrap3/locale/ar_IQ/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University Library
```

Comment:
@vormia, can you change 2017 to 2019 on this line and the next, and change "Simon Fraser University Library" to "Simon Fraser University"?

Reply:
Done